arcgis-ios-sdk-samples/Layers/Manage sublayers/ManageSublayers.storyboard

```diff
@@ -60,72 +60,57 @@
             <point key="canvasLocation" x="1039.2" y="160.56971514242881"/>
         </scene>
         <!--Sublayers-->
-        <scene sceneID="CRF-ji-nlo">
+        <scene sceneID="hgS-yY-ZCr">
             <objects>
-                <viewController automaticallyAdjustsScrollViewInsets="NO" id="JeC-e0-SwS" customClass="MapImageSublayersVC" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
-                    <view key="view" contentMode="scaleToFill" id="ank-tO-olW">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                <tableViewController id="Cix-X6-4BY" customClass="MapImageSublayersVC" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" allowsSelection="NO" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" id="7uH-wp-jAp">
```

Comment:
Thank you for converting this to a table view controller. Please use the grouped table view style.